src/haxelib/client/Main.hx

```diff
@@ -145,6 +145,7 @@ class Main {
 		apiVersion : "3.0",
 	};
 	static var IS_WINDOWS = (Sys.systemName() == "Windows");
+	static var PATH_SEPARATOR = if (IS_WINDOWS) ";" else ":";
```

Comment:
Right, forgot that you couldn't use `:` on windows.
In that case following perl/java/node makes sense.